backend/server/tsconfig.json

```diff
@@ -1,23 +1,24 @@
 {
   "compilerOptions": {
     "target": "ESNext",
-    "module": "NodeNext",
-    "rootDir": "src",
-    "allowJs": true,
+    "module": "CommonJS",
```

Comment:
Got some error messages due to how tsoa imports the files in the generated files. Hence why I changed this.